{"name": "困人处置列表-待处理", "status": "passed", "attachments": [{"name": "log", "source": "a2dd6374-2afe-4cd4-b799-1d372e7b9ae3-attachment.txt", "type": "text/plain"}], "start": 1692689153286, "stop": 1692689153507, "uuid": "252648bc-a400-4c4c-a1f6-71b16c4ba7a9", "historyId": "268ac27d44efa67b57ad425fd8b2df94", "testCaseId": "7335a6c89350efea2656ff5aefa50509", "fullName": "opsPC.test_emergency#test_device_error_trapped_list_002", "labels": [{"name": "story", "value": "应急处置"}, {"name": "feature", "value": "运维管理平台"}, {"name": "parentSuite", "value": "opsPC"}, {"name": "suite", "value": "test_emergency"}, {"name": "host", "value": "cale"}, {"name": "thread", "value": "13204-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "opsPC.test_emergency"}]}